### PR TITLE
Re-enabled monopole damage (woops)

### PR DESCRIPTION
**Pull request by Th3-spook**
I honestly have no idea how it happened but the end of life damage on the monopole was turned off (lmao)

### CODE REVIEW

File: Weapon Mods/Stable/Starcore_Serpent_Arms_Heavy_Metal/Data/Scripts/CoreParts/Spinal_Rotary_Ammo.cs [hunk 1]
```diff
@@ -132,7 +132,7 @@ partial class Parts
                     Armor = -1f, // Multiplier for damage against all armor. This is multiplied with the specific armor type multiplier (light, heavy).
                     Light = -1f, // Multiplier for damage against light armor.
                     Heavy = -1f, // Multiplier for damage against heavy armor.
-                    NonArmor = -1f, // Multiplier for damage against every else.
+                    NonArmor = -.5f, // Multiplier for damage against every else.
```
Comment (Jnick-24) [inline Weapon Mods/Stable/Starcore_Serpent_Arms_Heavy_Metal/Data/Scripts/CoreParts/Spinal_Rotary_Ammo.cs:135]: lol?

Comment (Th3-spook) [reply Weapon Mods/Stable/Starcore_Serpent_Arms_Heavy_Metal/Data/Scripts/CoreParts/Spinal_Rotary_Ammo.cs:135]: wait

Comment (Th3-spook) [reply Weapon Mods/Stable/Starcore_Serpent_Arms_Heavy_Metal/Data/Scripts/CoreParts/Spinal_Rotary_Ammo.cs]: man

**Review by Jnick-24 — APPROVED** [review]
yeah I'm thinking real

**Pull request merged**